### PR TITLE
Converters

### PR DESCRIPTION
**Pull request by richrines1**
* add `converters.py` with str <--> bytes conversions

### CODE REVIEW

File: applications_superstaq/qubo.py [hunk 1]
```diff
@@ -12,7 +14,7 @@ def read_json_qubo_result(json_dict: dict) -> np.recarray:
     Returns:
         a numpy.recarray containing the results of the optimization.
     """
-    return np.loads(codecs.decode(json_dict["solution"].encode(), "base64"))
```
Comment (singular-value) [inline applications_superstaq/qubo.py:15]: nice this fixes a warning too :)

**Review by singular-value — APPROVED** [review]
lgtm

**Review by singular-value — APPROVED** [review]
lgtm!

**Review by vtomole — COMMENTED** [review]
Deferring review to @singular-value

**Pull request merged**